src/ert/config/parsing/config_schema_deprecations.py

```diff
@@ -177,4 +177,25 @@
         "for the Ensemble Smoother update algorithm. "
         "Please use ENKF_ALPHA and STD_CUTOFF keywords instead.",
     ),
+    DeprecationInfo(
+        keyword="QUEUE_OPTION",
+        message="The LSF_LOGIN_SHELL queue option has been removed and no longer has any "
+        "effect. It has been used in the past to force the bsub command to use a "
+        "specific shell. The current ERT default is to use local shell.",
+        check=lambda line: "LSF_LOGIN_SHELL" in line,
+    ),
+    DeprecationInfo(
+        keyword="QUEUE_OPTION",
+        message="The LSF_RSH_CMD queue option has been removed and no longer has any "
+        "effect. It has been used in the past to set the remote shell command. "
+        "The ERT default is to use /usr/bin/ssh.",
+        check=lambda line: "LSF_RSH_CMD" in line,
+    ),
+    DeprecationInfo(
+        keyword="QUEUE_OPTION",
+        message="The LSF_SERVER queue option has been deprecated as it is already "
+        "being configured by ert-configurations. You should only use this queue option "
```

Comment:
We should not refer to `ert-configurations` in the ert source code, as outside of Equinor, "ert-configurations" does not exist.

Just say it is deprecated because it will become a no-op in the future